src/yunohost/settings.py

```diff
@@ -296,6 +297,32 @@ def reconfigure_ssh(setting_name, old_value, new_value):
         service_regen_conf(names=['ssh'])
 
 @post_change_hook("security.postfix.compatibility")
-def reconfigure_ssh(setting_name, old_value, new_value):
+def reconfigure_postfix(setting_name, old_value, new_value):
     if old_value != new_value:
         service_regen_conf(names=['postfix'])
+
+@post_change_hook("pop3.enabled")
+def reconfigure_dovecot(setting_name, old_value, new_value):
+    dovecot_package = 'dovecot-pop3d'
+
+    environment = os.environ.copy()
+    environment.update({'DEBIAN_FRONTEND': 'noninteractive'})
+
+    if new_value == "True":
```

Comment:
Is it really `"True"` that we want here and not `True` ? (idk)

Reply:
That's what my `ipdb` session tells me so that is what I check for.